management/src/test/java/nearsoft/skytouch/management/TaskApplicationTests.java

```diff
@@ -0,0 +1,16 @@
+package nearsoft.skytouch.management;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.context.junit4.SpringRunner;
+
+@RunWith(SpringRunner.class)
+@SpringBootTest
+public class TaskApplicationTests {
+
+    @Test
+    public void contextLoads() {
```

Comment:
Just add Tests when they really test something. If they doesn't have any useful content for the application there's no need to add them to the PR. 

Be careful about it.

Reply:
Ok thanks!

Reply:
will you remote this then?

Reply:
i removed the tests for this module for now later when i finish the microservice module i will add some tests for this